packages/menu/src/submenu.vue

```diff
@@ -226,6 +229,9 @@
         this.referenceElm = this.$el;
         this.popperElm = this.$refs.menu;
         this.updatePlacement();
+      },
+      close() {
+        if (this.opened) this.rootMenu.closeMenu(this.index);
```

Comment:
刚才试了一下，把这个放在 nextTick 中会不会更好一点？
```suggestion
        this.$nextTick(() => {
          if (this.opened) this.rootMenu.closeMenu(this.index);
        });
```